Configuration/HLT/python/autoCondHLT.py

```diff
@@ -54,4 +64,14 @@ def autoCondHLT(autoCond):
            autoCond[key] = ( autoCond[val[0]] )
         else:
            autoCond[key] = ( autoCond[val[0]],) + val[1]
+#
+    for key in ('run2_mc', 'run2_data', 'run2_data_relval', 'run2_hlt', 'run2_hlt_relval'):
+        old = autoCond[key]
+        autoCond[key] = ( old, ) + l1Menus['GRun']
+
+    for key in ('run2_mc_hi', 'run2_hlt_hi'):
+        old = autoCond[key]
+        autoCond[key] = ( old, ) + l1Menus['HIon']
+
```

Comment:
@mmusich 
Thanks a lot! 
Does that mean for TSG usage we should customize ```run2_data_relval``` and ```run2_hlt_relval``` in our own autoCondHLT.py file above, rather than ```run2_data``` and ```run2_hlt```?